src/js/libpannellum.js

```diff
@@ -1196,8 +1196,8 @@ function Renderer(container) {
         TextureImageLoader.prototype.loadTexture = function(src, texture, callback) {
             this.texture = texture;
             this.callback = callback;
-            this.image.src = globalParams.imageSrcFormatter 
-                                ? globalParams.imageSrcFormatter(src)
+            this.image.src = (globalParams.imageSrcAuthKeyFunc && image.queryStrings)
+                                ? src + '?' + image.queryStrings[globalParams.imageSrcAuthKeyFunc(src)]
```

Comment:
why not pass in the image into the imageSrcFormatter, too?
then you can do the lookup inside of it. That feels like a cleaner interface then pannellum internals needing to be aware of another field that you pass in via the image config?